src/devices/tuya.ts

```diff
@@ -1685,6 +1685,81 @@ const definitions: Definition[] = [
             {vendor: 'Moes', model: 'MS-105-M'},
         ],
     },
+    {
+        fingerprint: [
+            {
+                modelID: 'TS0601',
+                manufacturerName: '_TZE204_5cuocqty',
+            },
+        ],
+        model: 'ZDMS16-1',
+        vendor: 'Avatto',
+        description: 'Zigbee 1 channel Dimmer',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        configure: tuya.configureMagicPacket,
+        exposes: [
+            tuya.exposes.lightBrightnessWithMinMax(),
+            tuya.exposes.countdown(),
+            tuya.exposes.switchType(),
+            e.power_on_behavior().withAccess(ea.STATE_SET),
+        ],
+        meta: {
+            tuyaDatapoints: [
```

Comment:
This one looks to be the same as `TS0601_dimmer_5`?

Reply:
same as before, it's not. the new one doesn't support lightType and uses endpoint 4 for switchType.
also the switchType enum has a different mapping.